.devcontainer/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/devcontainers/python:3.11-bullseye
+
+RUN apt-get update && \
+    apt-get install -y python3-venv git curl && \
+    rm -rf /var/lib/apt/lists/*
+
+# Copy the devcontainer folder first
+COPY . /workspaces/py-launch-blueprint-main/.devcontainer
+
+# Then copy the rest of the repo (excluding .devcontainer)
+COPY .. /workspaces/py-launch-blueprint-main
+
+WORKDIR /workspaces/py-launch-blueprint-main
+
+# Make sure the setup script is executable
+RUN chmod +x /workspaces/py-launch-blueprint-main/.devcontainer/setup.sh
```

Comment:
@Tatianah3 Can you check if this is a necessary step? I'm uncertain if you copy a file into the Docker if it'll retain the permissions or not. That I think about it's probably not, but please verify. You can do that by trying to copy it without setting the permissions and see if it works.